features/donation.feature

```diff
@@ -24,6 +24,15 @@ Feature: The goal is to donate one time or multiple time with a subscription
       | /campus       |
       | /articles     |
 
+  Scenario: A user can't donate more than 7500€ per year
```

Comment:
Not true anymore

Reply:
It's true if we doesn't care of subscriptions, no ?

Reply:
Sorry I forgot to remove this comment ^^'